cfgov/jinja2/v1/_layouts/content-base-sidebar-first.html

```diff
@@ -3,7 +3,10 @@
 {% block pre_content scoped -%}
     {# TODO: Remove non-wagtail logic once old pages have been converted #}
     {% if page %}
+        {% if not breadcrumb_items %}
         {% set breadcrumb_items = page.get_breadcrumbs(request) %}
+        {% endif %}
+        
```

Comment:
TRIVIAL: remove this space or add it to content-base-main-first so that their same code blocks are identical.